### PR TITLE
Add foreground mode

### PR DESCRIPTION
**Pull request by naterini**
nsca has issues working with supervisord and other process managers used in docker containers. This patch stops nsca from forking which allows it to monitored and be run by a process manager.

This patch is based on https://github.com/efouts/nagios-nsca-supervisor/commit/5507503659c1c7ac47544b29be425a59bfb25c11

### CODE REVIEW

**Review by hedenface — APPROVED** [review]
Thanks! Looks good to me.

**Review by hedenface — APPROVED** [review]
This was okay, but the merge conflict was a bit painful. We push staging changes into maint branch for next release.

Thanks!

File: src/nsca.c [hunk 1]
```diff
@@ -128,7 +130,7 @@ int main(int argc, char **argv){
                 printf("This program is designed to accept passive check results from\n");
                 printf("remote hosts that use the send_nsca utility.  Can run as a service\n");
                 printf("under inetd or xinetd (read the docs for info on this), or as a\n");
-                printf("standalone daemon.\n");
+                printf("standalone daemon or forground process.\n");
```
Comment (xavierba) [inline src/nsca.c:133]: typo above, s/forground/foreground/

File: src/send_nsca.c [hunk 2]
```diff
@@ -455,7 +456,7 @@ int process_arguments(int argc, char **argv){
 			else
 				return ERROR;
 		        }
-
+
```
Comment (xavierba) [inline src/send_nsca.c:473]: spurious whitespaces

**Pull request merged**